documentation/library_guide/cmake_support.rst

```diff
@@ -0,0 +1,108 @@
+
++--------------------+--------+--------+--------+
+| ONEDPL_PAR_BACKEND | oneTBB | OpenMP | Serial |
++====================+========+========+========+
+| [not set]          |     oneDPL heuristics    |
++--------------------+--------+--------+--------+
+| tbb                |   X    |        |        |
++--------------------+--------+--------+--------+
+| openmp             |        |    X   |        |
++--------------------+--------+--------+--------+
+| serial             |        |        |    X   |
++--------------------+--------+--------+--------+
+
+The |onedpl_short| heuristics are the following: the first suitable backend is chosen among ``oneTBB``, ``OpenMP`` and ``Serial``, in that order.  If ``ONEDPL_PAR_BACKEND`` is specified, but the selected backend is not found or unsupported, |onedpl_short| is considered not found (``oneDPL_FOUND=False``).
+
+Backend for Device Execution Policies
+-----------------------------------------------------------
+The |onedpl_short| backend for device execution policies controls if device policies are enabled.
+
++-------------------+
+|       DPCPP       |
++===================+
+| oneDPL heuristics |
++-------------------+
+
+The heuristics are the following: ``DPCPP`` backend is enabled if the compiler supports -fsycl option and SYCL headers are available.
+
+For more details on |onedpl_short| backends, see :doc:`Execution Policies <parallel_api/execution_policies>`.
+
+Example CMake File
+==================
+To use |onedpl_short| with CMake, a user must create a ``CMakeLists.txt`` file for their project and add |onedpl_short|.  This file should be placed in the project's base directory.  Below is an example ``CMakeLists.txt`` file:
+
+.. code:: cpp
+
+  if (CMAKE_HOST_WIN32)
+    find_package(oneDPLWindowsIntelLLVM)
+  endif()
+
+  project(Foo)
+  add_executable(foo foo.cpp)
+  
+  # Search to find oneDPL
+  find_package(oneDPL REQUIRED)
+  
+  # Connect oneDPL to foo
+  target_link_libraries(foo oneDPL)
+
+**Note:** On Windows, some workarounds may be required to use icx[-cl] successfully with |onedpl_short|.  We recommend updating to the most recent version of CMake to minimize the workarounds required for successful use.  A CMake package has been provided, ``oneDPLWindowsIntelLLVM``, to provide the necessary workarounds to enable support for icx[-cl] on Windows with CMake versions 3.20 and greater.  Some workarounds are provided for icpx, but it is not fully supported on Windows at this time.  To use this package, please add ``find_package(oneDPLWindowsIntelLLVM)`` to your CMake file *before* you call ``project()``.
+
+
+Example CMake Invocation
+========================
+After creating a ``CMakeLists.txt`` file for their project, a user may use a command line CMake invocation to generate build scripts for their project.
```

Comment:
Please change "a user" to "you" and "their" to "your".

Reply:
Fixed these throughout, thanks.